js/modules/sunburst.src.js

```diff
@@ -160,46 +160,45 @@ var getDlOptions = function getDlOptions(params) {
         {})[0], options = merge({
         style: {}
     }, optionsLevel, optionsPoint), rotationRad, rotation, rotationMode = options.rotationMode;
-    if (options.textPath) {
-        if (point.shapeExisting.innerR === 0 &&
-            optionsLevel.textPath &&
-            optionsLevel.textPath.enabled) {
-            // Center dataLabel - disable textPath
-            options.textPath.enabled = false;
-            // Setting width and padding
-            options.style.width = Math.max(((point.shapeExisting.r * 2)) -
-                2 * (options.padding || 0), 1);
-        }
-        else if (point.dlOptions &&
-            point.dlOptions.textPath &&
-            !point.dlOptions.textPath.enabled &&
-            optionsLevel.textPath.enabled) {
-            // Bring dataLabel back if was a center dataLabel
-            options.textPath.enabled = true;
-        }
-        // Rest of the dataLabels
-        if (options.textPath.enabled) {
-            // Setting width and padding
-            options.style.width = Math.max(((point.outerArcLength +
-                point.innerArcLength) / 2) -
-                2 * (options.padding || 0), 1);
-        }
-    }
-    else if (!isNumber(options.rotation)) {
-        if (rotationMode === 'auto') {
+    if (!isNumber(options.rotation)) {
+        if (rotationMode === 'auto' || rotationMode === 'circular') {
```

Comment:
`rotationMode === 'circular'` is in a few places, consider to extract this as variable.